components/director/cmd/main.go

```
@@ -158,3 +165,18 @@ func configureLogger() {
 	})
 	log.SetReportCaller(true)
 }
+
+func getTenantMappingHanderFunc(transact persistence.Transactioner, staticUsersSrc string) func(writer http.ResponseWriter, request *http.Request) {
+	if v := os.Getenv("TENANT_MAPPING_MOCK"); v != "" {
+		return tenantmapping.NewHandler().ServeHTTP
+	}
+
+	uidSvc := uid.NewService()
+	authConverter := auth.NewConverter()
+	systemAuthConverter := systemauth.NewConverter(authConverter)
+	systemAuthRepo := systemauth.NewRepository(systemAuthConverter)
+	systemAuthSvc := systemauth.NewService(systemAuthRepo, uidSvc)
+	staticUsersRepo, _ := tenantmapping.NewStaticUserRepository(staticUsersSrc)
```

why the error is ignored?